prow/cmd/branchprotector/protect.go

```diff
@@ -47,9 +47,12 @@ type options struct {
 	config             configflagutil.ConfigOptions
 	confirm            bool
 	verifyRestrictions bool
-	tokens             int
-	tokenBurst         int
-	github             flagutil.GitHubOptions
+
+	// TODO(petr-muller): Remove after August 2021, replaced by github.ThrottleHourlyTokens
```

Comment:
Maybe add a deprecation warning to ANNOUNCEMENTS.md?

Reply:
Sounds good!